A36-xds-for-servers.md

```diff
@@ -95,23 +136,27 @@ The `GRPC_XDS_BOOTSTRAP` file will be enhanced to have a new field:
 
 XdsServer will use the normal `XdsClient` to communicate with the xDS server.
 There is no default value for `server_listener_resource_name_template` so if it
```

Comment:
Didn't we say that it was going to default to `%s`?

Reply:
Okay.  This means that users basically need to add this to their bootstrap files before they can use xDS with a gRPC server, but I guess that's okay.